generation/um/wslapi/generate.rsp

```diff
@@ -0,0 +1,22 @@
+@../../settings.rsp
+@../../remap.rsp
+--exclude
+operator^
```

Comment:
TerraFX is generally doing exact match exclusions to help clarify what is being dropped. If you add `--config log-exclusions` it will give you the full method name of anything that was partially matched.